From e0acc0e285988e144fcc86f1bb2e048b495b1f24 Mon Sep 17 00:00:00 2001
From: David Sniatkiwicz <davidsn@marvell.com>
Date: Thu, 6 Jul 2017 15:19:38 +0300
Subject: [PATCH 1083/1345] fix: doc: mvebu: a80x0: fix setup B incorrect SW1
 configuration

commit  ffdcd8fca94f37f05b403f6effe51c95af628d9e from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Current SW1 documentation is incorrect,
PCIe X4 is not functional when SW1 is configured according
to the documentation.

This patch fixes SW1 documentation.

Change-Id: I09473ea125329478b1f3a9d5deabd82726c879cf
Signed-off-by: David Sniatkiwicz <davidsn@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/41263
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../mvebu/a7k-a8k/armada8040-db-setup.txt          |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Documentation/mvebu/a7k-a8k/armada8040-db-setup.txt b/Documentation/mvebu/a7k-a8k/armada8040-db-setup.txt
index d35fc63..13caa51 100644
--- a/Documentation/mvebu/a7k-a8k/armada8040-db-setup.txt
+++ b/Documentation/mvebu/a7k-a8k/armada8040-db-setup.txt
@@ -30,9 +30,9 @@ For A8040 A0 revision SoC - Please use armada-8040-db-A0 configuration
 |---------------|-----------------------|-----------------------|-----------------------|-----------------------|-----------------------|
 |Device tree	|armada-8040-db-A.dtb	|armada-8040-db-B.dtb	|armada-8040-db-C.dtb	|armada-8040-db-D.dtb	|armada-8040-db-E.dtb	|
 |CP0-SW1:1	| OFF			| OFF			| OFF			| OFF			| OFF			|
-|CP0-SW1:2	| OFF			| OFF			| OFF			| OFF			| OFF			|
-|CP0-SW1:3	| OFF			| OFF			| OFF			| OFF			| OFF			|
-|CP0-SW1:4	| OFF			| OFF			| OFF			| OFF			| OFF			|
+|CP0-SW1:2	| OFF			| ON			| OFF			| OFF			| OFF			|
+|CP0-SW1:3	| OFF			| ON			| OFF			| OFF			| OFF			|
+|CP0-SW1:4	| OFF			| ON			| OFF			| OFF			| OFF			|
 |CP0-SW1:5	| OFF			| OFF			| OFF			| OFF			| OFF			|
 |CP0-SW1:6	| OFF			| OFF			| OFF			| OFF			| OFF			|
 |CP0-SW1:7	| OFF			| OFF			| OFF			| OFF			| OFF			|
-- 
1.7.9.5

